botcore/utils/channel.py

```diff
@@ -0,0 +1,35 @@
+"""Useful helper functions for interacting with various discord.py channel objects."""
+
+import discord
+from discord.ext.commands import Bot
+
+from botcore.utils import loggers
+
+log = loggers.get_logger(__name__)
+
+
+def is_in_category(channel: discord.TextChannel, category_id: int) -> bool:
+    """
+    Return whether the given channel has the category_id.
```

Comment:
Yea, we also have two options, single backticks italicise, double backticks put into a code style EG:
![image](https://user-images.githubusercontent.com/9753350/155558237-77c1b1ae-a669-42a6-b551-e516a524c8f4.png)

Reply:
I'm going to go with double back ticks, since I think the code format is easier to differentiate. We can discuss if we want to change this after the PR.

Reply:
While I did bring up single vs double ticks in another comment, this comment was focused on when they should be applied rather than how.

Reply:
Yea that's true, I think as is in this PR I've gone for wrapping in backticks anywhere a param is mentioned in the docstring. Whether this is correct I'm not sure. As you said it's something we can look into after this PR.

Reply:
https://github.com/python-discord/bot-core/pull/29/commits/d7841816581737b4a7cece5bba98328a44630d4b